baseline-error-prone/build.gradle

```diff
@@ -13,3 +13,11 @@ dependencies {
 
     processor 'com.google.auto.service:auto-service'
 }
+
```

Comment:
this can be dropped once we release this change and pickup new baseline in baseline